packages/main/test/specs/MessageView.spec.js

```diff
@@ -0,0 +1,69 @@
+import { assert } from "chai";
+
+describe("MessageView general interaction", () => {
+    before(async () => {
+        await browser.url(`test/pages/MessageView.html`);
+    });
+
+    it("Segmented button should shown correct count of any item type", async () => {
+        // Arrange
+        const messageView = await browser.$("#mv2");
+        const headerButtons = await messageView.shadow$$(
+            "ui5-segmented-button-item"
+        );
+
+        // Assert
+        assert.strictEqual(headerButtons.length, 3);
+
+        assert.strictEqual(await headerButtons[0].getText(), "All");
```

Comment:
add assert success descriptions, because when a test fails one would not know from where exactly is the failure in the console

Reply:
Added description text to the strictEqual asserts